paper-api/src/main/java/io/papermc/paper/world/biome/effects/AdditionSound.java

```diff
@@ -0,0 +1,26 @@
+package io.papermc.paper.world.biome.effects;
+
+import org.bukkit.Sound;
+import org.jetbrains.annotations.Range;
+
+/**
+ * Settings for additions sound, which can play along
+ * the biome music and provides extra ambience to the biome.
+ */
+public interface AdditionSound {
+
+    /**
+     * One sound event — The additions sound.
```

Comment:
"One sound event" in these three classes reads kind of weird, it seems better without to me.